docs/examples/Parameters/Complex_Parameters.ipynb

```diff
@@ -67,7 +67,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `ComplexNumbers` validator requires explicitly complex values. Floats and ints will *not* pass."
+    "The `ComplexNumbers` validator requires explicitly complex values. `Floats` and `ints` will *not* pass."
```

Comment:
... in case you'd like to refer to actual python types
```suggestion
    "The `ComplexNumbers` validator requires explicitly complex values. `float`s and `int`s will *not* pass."
```